client/my-sites/stats/site.jsx

```diff
@@ -329,6 +330,7 @@ class StatsSite extends Component {
 								/>
 							)
 						}
+						<StatsSubscribers siteId={ siteId } />
```

Comment:
Blocking: Could we wrap this in a feature flag, as we do in line 336?

```suggestion
					{ config.isEnabled( 'stats/subscribers-section' ) && (
						<StatsSubscribers siteId={ siteId } />
					) }
```

Reply:
Pushed this change just before you commented 😄 All resolved now :)